day_6/exercises/person.js

```diff
@@ -8,4 +8,24 @@ person methods below the class so that they print their result to the
 terminal.
 */
 
-// YOUR CODE HERE
+class Person {
+ constructor (name, age) {
+this.name = name;
+this.age = age;
+this.learn = true
+ }
+ stressOut () {
+
+   console.log("AHHHHHHRGGGH!")
```

Comment:
How would you change this so that it updates the data model, instead of just logging to the console?